test-framework-docker/scheduler/src/main/java/com/containersolutions/mesoshelloworld/scheduler/FrameworkScheduler.java

```diff
@@ -62,29 +73,27 @@ public void resourceOffers(SchedulerDriver driver, List<Offer> offers) {
 
     @Override
     public void offerRescinded(SchedulerDriver driver, OfferID offerId) {
+        // not supported in tests. It's ok to get task rejected
     }
 
     @Override
     public void statusUpdate(SchedulerDriver driver, TaskStatus status) {
 
         System.out.println("Status update: task " + status.getTaskId().getValue() +
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS106)